recipes/medical_imaging.ipynb

```diff
@@ -1,112 +1,181 @@
 {
```

Comment:
<p>is there a reason why we now want to insist in the title on mask conversion rather than medical imagery ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Well the dicom -> jpeg conversion is not that challenging, it is already covered in the pydicom doc

What's interesting IMO in this tutorial is the kili label -> numpy mask conversion, which is easier to do now thanks to the kili.utils.labels helpers. That's why the title emphasizes on this now

Reply:
I agree, but according to me, the core of the notebook is to show how Kili can handle medical imagery. Mask conversion is an important step at the end but we also show that we can easily work and import dicom images with kili and gives some snippets to do it.
May be to check with PMs but I would be surprised that we want to move the core of the title of this notebook

Reply:
Well Kili does not handle dicom images, there is a conversion step to JPEG before

I'll check with Damien and @xczuba  yep!

Reply:
We changed the title to "How to upload medical images to Kili, and visualize segmentation labels with matplotlib"

We also put the tutorial in a new mkdocs section:

```yaml
- Manipulating Labels:
    - Converting and Visualizing Segmentation Labels: sdk/tutorials/medical_imaging.md
```

We will add the label parsing tutorials in this section too

Reply:
@xczuba and I are ok with this title yep: ""How to upload medical images to Kili, and visualize segmentation labels with matplotlib""

Reply:
good for me, should we wait for @xczuba to review the PR ?

Reply:
he's already done it, I merge 🚀